src/test/java/com/google/sps/SearchServletTest.java

```diff
@@ -151,4 +152,34 @@ public void getKeywordsHasMax5() throws IOException {
     List<String> testList = SearchServlet.getKeywords(titleText, descText);
     assertEquals(correctList, testList);
   }
+
+  @Test
+  public void getDistanceWorks() throws IOException {
+    int distance =
+        SearchServlet.getDistance(
+            new LatLng(-31.9522, 115.8589), new LatLng(-25.344677, 131.036692));
+    assertEquals(2059, distance);
+  }
```

Comment:
Does the tests send requests to the real Google Maps API? If so, would the tests fail if the driving route returned by Google Maps change?